### ISSUE TITLE
Leader CSS Polyfill 

### ISSUE THREAD

**Issue by eduarevalo** [pagedjs/pagedjs#147]
Hello team. Has anyone envision to write a leader css polyfill? 
Pagedjs have you guys explored that path? How complex can it be?

**Comment by julientaq** [comment]
so the problem with leader is that it’s hard to follow them.

sorry. i couldn’t resist.

so the leader in css is kind of a shortcut to something you can build with css and the proper html.
since the web evolved quite a lot since the specs, i think it would be quite hard to implement an universal leader system, 

but you can see how we did it for atla:

https://gitlab.coko.foundation/pagedjs/pagedjs-templates/atla/-/blob/master/src/css/modules/page-toc.css

this is how it looks

https://books.atla.com/atlapress/catalog/view/39/53/300 (page5)

the html looks like this: 

```
<ol class="toc">
     <li class="toc-body toc-part">
        <a href="#comp-number-931b7756-a268-40f1-b897-ab4c9b50436e">
            <span class="name">Building a case for open data metrics</span>
        </a>
     </li>
</ol>

```

tell me if that helps.
